### PR TITLE
Support addStyleTag and addScriptTag

### PR DESCRIPTION
**Pull request by paresharma**
This PR adds support for addStyleTag and addScriptTag.

Also, My humble apologies for not attending to the comments in the original PR #71.

### CODE REVIEW

**Comment by paresharma** [comment]
The failing spec seems random.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Unreleased
-- None
+- [#106](https://github.com/Studiosity/grover/pull/106) Add support for addStyleTag and addScriptTag
```
Comment (abrom) [inline CHANGELOG.md:4]: Don't forget to reference yourself. See L16 for an example. Then make sure you add yourself at the bottom of the file ;)

File: README.md [hunk 2]
```diff
@@ -171,6 +171,32 @@ And give that array to Grover:
...
+
+#### Adding script tags
+To add script tags, pass an array of script tag options as such
+See [page.addScriptTag](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageaddscripttagoptions) documentation for more details.
+
+```ruby
+script_tag_options = [
+  { url: 'http://example.com/script.js' },
+  { path: 'script.js' },
+  { content: 'document.querySelector('h1').style.display = "none"' }
```
Comment (abrom) [inline README.md:195]: Can you fix up your example? There are unescaped single quotes around the `h1`. Suggest wrap the `h1` with double quotes instead.

**Review by abrom — CHANGES_REQUESTED** [review]
Change looks good @paresharma 👍 

Just a few bits of cleanup needed ;-)

File: README.md (hunk 2 above)
Comment (paresharma) [reply README.md]: Thanks, updated.

File: CHANGELOG.md [hunk 3]
```diff
@@ -237,4 +237,4 @@
 [@braindeaf]: https://github.com/braindeaf
 [@ckhall]: https://github.com/ckhall
 [@anamba]: https://github.com/anamba
```
Comment (abrom) [inline CHANGELOG.md:239]: Can you please also add a reference to yourself here (that's what the link at the top will point to!)

Comment (paresharma) [reply CHANGELOG.md:239]: Oh yeah my bad. I din't notice this

**Review by abrom — APPROVED** [review]
Nice one, thanks @paresharma 👍

**Pull request merged**